app/src/main/java/com/phivle/fitnesstasks/fragments/CalendarFragment.java

```diff
@@ -0,0 +1,61 @@
+package com.phivle.fitnesstasks.fragments;
+
+import android.net.Uri;
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.phivle.fitnesstasks.R;
+
+/**
+ * A simple {@link Fragment} subclass.
+ * Activities that contain this fragment must implement the
+ * {@link CalendarFragment.OnFragmentInteractionListener} interface
+ * to handle interaction events.
+ */
+public class CalendarFragment extends Fragment {
+
+    private OnFragmentInteractionListener mListener;
+
+    public CalendarFragment() {
+        // Required empty public constructor
+    }
+
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
+        // Inflate the layout for this fragment
```

Comment:
this comment seems unnecessary

Reply:
I will remove all the comments. These comments were auto-generated when added fragments.